app/model/SubscriptionData.scala

```diff
@@ -96,3 +97,9 @@ object PersonalData {
 case class SubscribeRequest(genericData: SubscriptionData, productData: Either[PaperData, DigipackData]) {
   def productRatePlanId = productData.fold(_.plan.id, _.plan.id)
 }
+
+case class DeliveryRecipient(title: Option[Title], firstName: Option[String], lastName: Option[String], email: Option[String], address: Address) extends FullName {
+  val first = firstName.mkString
+  val last = lastName.mkString
+  val isGiftee = (Seq() ++ title.map(_.title) ++ firstName ++ lastName ++ email).exists(_.trim.nonEmpty)
```

Comment:
I don't understand why this confirms that someone is a giftee?

Reply:
I'll refactor this in a subsequent PR.